gensim/test/test_dtm.py

```diff
@@ -35,14 +36,35 @@ def setUp(self):
 
     def testDtm(self):
         if self.dtm_path is not None:
-            model = gensim.models.wrappers.DtmModel(self.dtm_path, self.corpus, self.time_slices, num_topics=2, id2word=self.id2word, model='dtm', initialize_lda=True)
+            model = gensim.models.wrappers.DtmModel(self.dtm_path, self.corpus, self.time_slices, num_topics=2,
```

Comment:
minor code style comment: we want to be indenting multi-line calls at normal indent levels. Not this at-opening-bracket-indent -- it creates too much visual clutter. Example:

```python
model = gensim.models.wrappers.DtmModel(
    self.dtm_path, self.corpus, self.time_slices, num_topics=2,
    id2word=self.id2word, model='dtm', initialize_lda=True,
    rng_seed=1)
```